### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by core-code**
since the numeric comparison isn't even in a single cask i suggest we don't document that it should be done

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Since the code to do it exists, I’d rather either remove that code first or keep it documented *but* have a warning that only the symbolic names are allowed in the official repos.

**Comment by core-code** [comment]
breaking 3rd party taps sounds like a bad idea, so option 2 sounds better?

**Comment by vitorgalvao** [comment]
> so option 2 sounds better?

Agreed. Less work, too.

File: doc/cask_language_reference/readme.md [hunk 1]
```diff
@@ -39,18 +39,13 @@ Tests on the following values are known to be acceptable:
...
 ```ruby
 if MacOS.version <= :mavericks     # symbolic name
 ```
 
-```ruby
-if MacOS.version <= '10.9'         # version string
-```
-
-The available symbols for macOS versions are: `:mavericks`, `:yosemite`, `:el_capitan`, `:sierra`, `:high_sierra`, and `:mojave`. The corresponding numeric version strings should be given as major releases containing a single dot.
+The available symbols for macOS versions are: `:mavericks`, `:yosemite`, `:el_capitan`, `:sierra`, `:high_sierra`, and `:mojave`.
```
Comment (miccal) [inline doc/cask_language_reference/readme.md]: Remove `:mavericks` :

https://github.com/Homebrew/homebrew-cask/issues/58046#issuecomment-460004001

Comment (core-code) [reply doc/cask_language_reference/readme.md]: thanks there. i've added catalina, but wasn't sure about removing. officially we only support sierra and up, so only removing mavericks seemed kinda arbritrary...

**Pull request merged**